Gems/PhysX/Code/Source/PrismaticJointComponent.cpp

```diff
@@ -84,4 +87,83 @@ namespace PhysX
             m_jointSceneOwner = leadFollowerInfo.m_followerBody->m_sceneOwner;
         }
     }
+
+    // AZ::Component
+    void PrismaticJointComponent::Activate() {
+        JointComponent::Activate();
+        PhysX::JointInterfaceRequestBus::Handler::BusConnect(this->GetEntityId());
+    };
+
+    void PrismaticJointComponent::Deactivate() {
+        JointComponent::Deactivate();
+        PhysX::JointInterfaceRequestBus::Handler::BusDisconnect();
+    };
+
+    physx::PxD6Joint* GetPhysXD6Joint(const AzPhysics::JointHandle& joint_handle,
+                                                        const AzPhysics::SceneHandle& joint_scene_owner)
+    {
```

Comment:
nit: Indentation looks a bit out here

Reply:
Reworked this function and fixed formatting.